src/modules/SideBar/SideBar.module.css

```diff
@@ -0,0 +1,6 @@
+@import '_colors.css';
+@import '_fonts.css';
+
+.side-bar {
+  width: 80%;
```

Comment:
why not 100% width?

Reply:
@VladSolodjankin 100% width looks wider than how it is in our design